system/jhtree/keybuild.cpp

```diff
@@ -278,7 +278,7 @@ class CKeyBuilder : public CInterfaceOf<IKeyBuilder>
 
     offset_t nextLevel()
     {
-        offset_t ret = endLevel(false);
+        endLevel(false);
```

Comment:
What is the return value supposed to be? Should we be returning ret?

Reply:
endLevel() function seems completely redundant, and nextLevel a bit pointless... if nextLevel always returns 0 then there's also some redundant code in buildTree.

Feels like this may be supporting some very long-deleted functionality